libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -466,6 +467,26 @@ bool AP_Arming::board_voltage_checks(bool report)
     return true;
 }
 
+bool AP_Arming::fence_checks(bool report)
+{
+    if ((checks_to_perform & ARMING_CHECK_ALL) ||
+        (checks_to_perform & ARMING_CHECK_FENCE)) {
```

Comment:
This processing is done when the ALL bit or FENCE bit is set in the setting value. I think that it is better to code this as a code.